### PR TITLE
Graph migration

### LINKED ISSUE #34: Update the developer documentation after the migration to TheGraph network
Also replace Kovan with Sepolia

https://github.com/kleros/kleros-docs/blob/ac2111dbbf0de399f382f59cce0330d47f95f5e1/developer/light-curate.md?plain=1#L35

Blocked by https://github.com/kleros/gtcr-subgraph/issues/25, https://github.com/kleros/gtcr-subgraph/issues/26
Resolves https://github.com/kleros/builder-baseline/issues/22

### PR DESCRIPTION
**Pull request by jaybuidl**

### CODE REVIEW

**Review by kemuru — COMMENTED** [review]
`REACT_APP_IPFS_GATEWAY=https://ipfs.kleros.io` should be updated to 'cdn.kleros.link'

> const url = `${process.env.REACT_APP_IPFS_GATEWAY}/add`

 outdated, refer to [court-functions's upload to ipfs function (pinFiles())](https://github.com/kleros/court-functions/blob/master/functions/upload-to-ipfs.ts#L59)

File: developer/classic.md [hunk 1]
```diff
@@ -148,7 +146,7 @@ const gtcr = new GeneralizedTCR(
   }
 ```
 
-Example [https://codesandbox.io/s/kind-banach-4fxnj](https://codesandbox.io/s/kind-banach-4fxnj)
+Example: [https://codesandbox.io/s/kind-banach-4fxnj](https://codesandbox.io/s/kind-banach-4fxnj)
```
Comment (tractorss) [inline developer/classic.md]: The codesandbox code fails to run

File: developer/light-curate.md [hunk 2]
```diff
@@ -247,22 +205,22 @@ const tx = await gtcr.addItem(ipfsEvidencePath, {
...
 
-Fetchin items is best done via the subgraph we provide. If you deployed an list using the factory, it already has a subgraph deployed and available \(here\)\[[https://thegraph.com/explorer/subgraph/kleros/light-curate-kovan](https://thegraph.com/explorer/subgraph/kleros/light-curate-kovan)\].
+Fetching items is best done via the subgraph we provide. If you deployed a list using the factory, it already has a subgraph deployed and available [here](https://thegraph.com/explorer/subgraphs/9hHo5MpjpC1JqfD3BsgFnojGurXRHTrHWcUcZPPCo6m8?view=Query&chain=arbitrum-one).
 
 #### List
 
 Whenever we want to fetch items, or a specific item, we must pass the TCR address to the subgraph.
 
-See \(this react example\)\[[https://github.com/kleros/gtcr/blob/5e313ced24f5e3fc3a54f812e07fb1f86a6b2621/src/pages/items/index.js](https://github.com/kleros/gtcr/blob/5e313ced24f5e3fc3a54f812e07fb1f86a6b2621/src/pages/items/index.js)\] for more details.
+See [this react example](https://github.com/kleros/gtcr/blob/5e313ced24f5e3fc3a54f812e07fb1f86a6b2621/src/pages/items/index.js) for more details.
```
Comment (tractorss) [inline developer/light-curate.md]: This code example points to an the pre-migration code, where the query is incorrect.
https://github.com/kleros/gtcr/blob/5e313ced24f5e3fc3a54f812e07fb1f86a6b2621/src/pages/items/index.js#L255C1-L257C16

**Review by tractorss — COMMENTED** [review]
made some changes , rest lgtm

**Review by gmkung — APPROVED** [review]
I didn't run the code myself but looking at the rendered markdown, LGTM

File: developer/light-curate.md (hunk 2 above)
Comment (jaybuidl) [reply developer/light-curate.md]: Updated

File: developer/classic.md (hunk 1 above)
Comment (jaybuidl) [reply developer/classic.md]: No idea what code was running there, removed the link.

**Pull request merged**